### PR TITLE
51 ensure tables pages are responsive

### PR DESCRIPTION
**Pull request by Miladepour**
## Description

Make all pages responsive 

## Related to

#51 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/Hero.js [hunk 1]
```diff
@@ -15,19 +15,15 @@ function Hero() {
           <Image  src="https://i.ibb.co/j6f02Js/kunji-1.jpg" alt="Right Column Image" fluid className={styles.heroImages} />
         </Col>
       </Row >
-    </Container>
-    <Container>
       <Row className={styles.rowMain}>
         <Col md={6}>
-          <Image src="https://i.ibb.co/HgTHb2j/kunji-2.jpg" alt="Right Column Image" fluid className={styles.heroImages} />
+          <Image src="https://i.ibb.co/HgTHb2j/kunji-2.jpg" alt="Left Column Image" fluid className={styles.heroImages} />
```
Comment (EneVladMihai) [inline client/src/components/Hero.js:20]: The alt is a textual alternative of the image - I.e. brief description of what is in the image

https://www.w3schools.com/TAGS/att_img_alt.asp

File: client/src/components/Hero.module.css [hunk 2]
```diff
@@ -9,8 +9,8 @@
     display: flex;
```
Comment (EneVladMihai) [inline client/src/components/Hero.module.css:9]: This is already true for 'Row' components.

File: client/src/components/Hero.module.css [hunk 3]
```diff
@@ -9,8 +9,8 @@
...
     flex-wrap: wrap;
```
Comment (EneVladMihai) [inline client/src/components/Hero.module.css:10]: This is already true for 'Row' components.

File: client/src/components/Hero.module.css [hunk 4]
```diff
@@ -9,8 +9,8 @@
...
     align-items: center;
```
Comment (EneVladMihai) [inline client/src/components/Hero.module.css:11]: Bootstrap has a utility class for it 'align-items-center'

https://getbootstrap.com/docs/4.0/utilities/flex/#align-items

File: client/src/components/Hero.module.css [hunk 5]
```diff
@@ -9,8 +9,8 @@
...
-    
+    flex-direction: row-reverse;
```
Comment (EneVladMihai) [inline client/src/components/Hero.module.css:12]: This makes the component code harder to read. Components have a certain order in the component file, but are reversed by a class definition which is a bit hidden in a style file.

File: client/src/components/UserForm.module.css [hunk 6]
```diff
@@ -7,4 +7,20 @@
   display: flex;
   justify-content: space-between;
   margin-bottom: 25px;
+}
+button[type="submit"] {
```
Comment (EneVladMihai) [inline client/src/components/UserForm.module.css:11]: I'd say this is a bit generic and makes the component code hard to read, I'd suggest using the `mt-x` class from bootstrap.

https://getbootstrap.com/docs/4.0/layout/utilities-for-layout/#margin-and-padding

**Review by EneVladMihai — CHANGES_REQUESTED** [review]
Some concerns about readability and introducing styling that is already available out of the box from bootstrap. I can not test the code as I'm on a phone device, but if it could be deployed I can have a look. (An alternative is to use sth like ngrok to 'host' it from your local computer for a short time)

File: client/src/components/Hero.js (hunk 1 above)
Comment (Miladepour) [reply client/src/components/Hero.js]: Done.

File: client/src/components/Hero.module.css (hunk 2 above)
Comment (Miladepour) [reply client/src/components/Hero.module.css]: Done.

File: client/src/components/Hero.module.css (hunk 4 above)
Comment (Miladepour) [reply client/src/components/Hero.module.css]: Done.

**Review by natyfromwonderland — APPROVED** [review]
Need to deploy and see how it looks on various mobile devices. This will show if any more tweaks are needed.

**Pull request merged**